README.rst

```diff
@@ -185,6 +185,14 @@ To run these tests:
    # run the python API tests:
    tox -e py_func_v4
 
+When developing tests it can be a little frustrating to wait for GitLab to spin
+up every run. To prevent the containers from being cleaned up afterwards, pass
```

Comment:
Could you just add a quick note that a manual docker-compose down/cleanup might be needed if the user wants a clean start after running this?

Reply:
Sure thing.